hack/build/run-functional-tests.sh

```diff
@@ -95,11 +95,21 @@ fi
 
 (
     export TESTS_WORKDIR=${CDI_DIR}/tests
-    ginkgo_args="--trace --timeout=8h"
+    ginkgo_args="--trace --timeout=8h --v"
+
+    if [[ -n "$CDI_E2E_SKIP" ]]; then
+        ginkgo_args="${ginkgo_args} --ginkgo.skip=${CDI_E2E_SKIP}"
```

Comment:
I think we can use --skip since we now supply it to the ginkgo cli, similarly to how you used --focus

Reply:
I see you resolved this btw?